proto/blueprint.proto

```diff
@@ -493,6 +499,12 @@ message Row {
   optional Thrasher thrasher = 5;
   RowType type = 6;
   optional string title = 7;
+
+  /**
+   * When this attribute is true, the spacing in between cards in the row is 
+   * reduced.
+   */
+  optional bool has_tight_spacing = 8;
```

Comment:
Nit: In the generated protobuf Swift models properties are generated with the prefix`has` to identify whether an optional property does indeed exist. 
Maybe it would be better to rename this to `tightenSpacing`?

Reply:
Sounds good to me!